apps/test/unit/templates/sectionProgress/progressTables/ProgressTableViewTest.jsx

```diff
@@ -97,6 +97,14 @@ describe('ProgressTableView', () => {
       .false;
   });
 
+  it('passes something down to child', () => {
```

Comment:
why "something" instead of "showSectionProgressDetails"?

Reply:
Oops! 🤦‍♀️ my brain wasn't working properly yet and I couldn't think of the name of the prop. I meant to go back and fill that in. Will do now